x509-cert/src/ext/pkix/sct.rs

```diff
@@ -0,0 +1,733 @@
+#![cfg(feature = "sct")]
+//! Signed Certificate Timestamp list extension as defined in the
+//! [Certificate Transparency RFC 6962].
+//!
+//! [Certificate Transparency RFC 6962]: https://datatracker.ietf.org/doc/html/rfc6962
```

Comment:
Following `std` we usually do comments first, then attributes...
```suggestion
//! Signed Certificate Timestamp list extension as defined in the
//! [Certificate Transparency RFC 6962].
//!
//! [Certificate Transparency RFC 6962]: https://datatracker.ietf.org/doc/html/rfc6962

#![cfg(feature = "sct")]
```